workers/run_tests.sh

```diff
@@ -34,6 +49,17 @@ cd $script_directory
 # move up a level
 cd ..
 
+# Ensure that Nomad is running first
```

Comment:
This is why all the tests are failing. Only the `foreman` tests require Nomad to be running because that's where we have our end-to-end tests. All of the `workers` tests mock out `send_job` so that we never actually hit Nomad. If you've seen anywhere where this isn't the case, I think the correct fix is to mock that function out.

The reason we do this is that in order for a local test Nomad agent to work we need a local Docker registry running which contains the images we want to use. Some of these images are large and pushing them to even a local registry is slow (thanks Docker) so we only do this in the circle tests once.